modules/standard/Random.chpl

```diff
@@ -576,20 +733,38 @@ module Random {
       /*
         Returns the next value in the random stream.
 
-        Reals are between 0 and 1; integers are in the range of the integer.
-        :returns: The next value in the random stream as type :type:`eltType`.
+        Generated reals are in [0,1] - both 0.0 and 1.0 are
+        possible values.
```

Comment:
Still leaves open some question about imag's, complex's.  Could say "floats" to try and dodge this, though even then complexes may deserve a little more explanation.